### PR TITLE
refactor: use FilterActivePods instead in replicaset controller

### PR DESCRIPTION
**Pull request by draveness**
/kind design

**What this PR does / why we need it**:

Use existing `FilterActivePods` instead

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/replicaset/replica_set.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ import (
 	"time"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (danielqsj) [inline pkg/controller/replicaset/replica_set.go]: why this change ?

Comment (draveness) [reply pkg/controller/replicaset/replica_set.go]: Sorry, my IDE automatically adds it to this file, already removed from this commit

**Comment by draveness** [comment]
Hi @danielqsj, what should I do now?

**Review by danielqsj — APPROVED** [review]
/lgtm

**Comment by danielqsj** [comment]
ping @caesarxuchao @lavalamp @enisoc for approval

**Comment by draveness** [comment]
ping @caesarxuchao @lavalamp @enisoc for approval 🤣
